src/Libraries/Microsoft.Extensions.Resilience/Resilience/Internal/ResilienceTagNames.cs

```diff
@@ -5,13 +5,13 @@ namespace Microsoft.Extensions.Resilience.Internal;
 
 internal static class ResilienceTagNames
 {
-    public const string FailureSource = "failure-source";
+    public const string FailureSource = "dotnet.resilience.failure.source";
 
-    public const string FailureReason = "failure-reason";
+    public const string FailureReason = "dotnet.resilience.failure.reason";
 
-    public const string FailureSummary = "failure-summary";
+    public const string FailureSummary = "dotnet.resilience.failure.summary";
 
-    public const string DependencyName = "dep-name";
+    public const string DependencyName = "dotnet.resilience.dependency.name";
```

Comment:
```suggestion
    public const string DependencyName = "dotnet.dependency.name";
```

The idea behind adding these tags to resilience metrics was that we can correlate resilience events with the http metrics. In that sense the name of this tag should be the same as the tag used to track `RequestName` and `DependencyName` for HTTP scenarios.

Do we know how the `RequestMetadata.DependencyName` and `RequestMetadata.RequestName` is represented in Otel world?

Reply:
In the current HttpClient metrics there's no such dimensions: https://learn.microsoft.com/en-us/dotnet/core/diagnostics/built-in-metrics-system-net#instrument-httpclientrequestduration
Neither OTel specifies this: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md#metric-httpclientrequestduration
I agree we need to unify these dimensions/attributes if they will represent same things in HttpClient metering (hopefully we'll add these bits eventually) and resilience libraries.

Reply:
Yup correct, these are not strictly tied to resilience so it would be better to not include it in the name to avoid doing breaking changes in the future.

Ideally, this would be the same name as for HTTP metrics.